idle_sweep_test.go

```diff
@@ -66,6 +66,16 @@ func (pl *peerStatusListener) waitForZeroConnections(t testing.TB, channels ...*
 	}
 }
 
+func (pl *peerStatusListener) waitForZeroExchanges(t testing.TB, channels ...*Channel) {
+	for i := 0; i < 10; i++ {
```

Comment:
consider `testutils.WaitFor`
https://github.com/uber/tchannel-go/blob/v1.22.2/testutils/wait.go#L30

it handles scaling timeouts by the scale factor, and does increasing wait times between attempts, etc.